wire-schema/src/main/java/com/squareup/wire/schema/Options.kt

```diff
@@ -256,12 +297,20 @@ class Options(
   }
 
   fun retainAll(schema: Schema, markSet: MarkSet): Options {
-    if (map == null || map!!.isEmpty()) return this // Nothing to prune.
+    if (entries.isNullOrEmpty()) return this // Nothing to prune.
```

Comment:
is this invoked in both null and empty cases?

Reply:
yes